src/lib.rs

```diff
@@ -214,13 +214,18 @@ pub use word_splitters::WordSplitter;
 pub mod wrap_algorithms;
 pub use wrap_algorithms::WrapAlgorithm;
 
+mod line_ending;
+pub use line_ending::LineEnding;
+
 pub mod core;
 
 /// Holds configuration options for wrapping and filling text.
 #[derive(Debug, Clone)]
 pub struct Options<'a> {
     /// The width in columns at which the text will be wrapped.
     pub width: usize,
+    /// TODO doc
```

Comment:
(Still need a docstring here.)

Reply:
ACK, sorry, missed it entirely

Reply:
All good, I just left comments so that we won't forget :-)